srcpkgs/mkinitcpio/files/69-dm-lvm.rules

```diff
@@ -0,0 +1,67 @@
+# Copyright (C) 2012,2021 Red Hat, Inc. All rights reserved.
```

Comment:
I copied the file from Arch Linux udev. It seems near enough from `69-dm-lvm-metad.rules`, without the `LVMetaD` bits

Reply:
a file with the same name is shipped with the `lvm2` package already. is it significantly different? mkinitcpio hooks can install udev rules from the system too

Reply:
a file with the same name is installed by `lvm2`. is it significantly different?